user/views.py

```diff
@@ -40,4 +41,11 @@ class UserProfileView(DetailView):
 
 
 class UserProfileEditView(UpdateView):
-    pass
+    model = User
+    template_name = 'user/update.html'
+    context_object_name = 'user'
+    form_class = UserUpdateForm
+
+    def get_success_url(self):
+        success_url = self.success_url = reverse_lazy('user:profile', args=[self.kwargs['slug']])
```

Comment:
Double equal is redundant here. Simply return redirect(reverse_lazy('user:profile', args=[self.kwargs['slug']]))